srcpkgs/ibus-bamboo/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'ibus-bamboo'
+pkgname=ibus-bamboo
+version=0.4.6
+revision=1
+archs=noarch
+build_style=go
+go_import_path="github.com/BambooEngine/ibus-bamboo"
+go_package="github.com/BambooEngine/ibus-bamboo/src/ibus-bamboo"
+hostmakedepends="git"
+makedepends="libXtst-devel libX11-devel"
+depends="ibus"
+short_desc="Vietnamese input method for ibus"
+maintainer="ndgnuh <ndgnuh99@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/BambooEngine/ibus-bamboo"
+distfiles="https://${go_import_path}/archive/v${version}.tar.gz>${pkgname}-${version}.tar.gz"
+checksum="2345394fd1fd8de5a9f620f3775ab34c3233c16eb6a9ffe166e3ed6f1a31f1bb"
+conflicts="ibus-bamboo-git"
```

Comment:
There is no `ibus-bamboo-git` package.

Reply:
I'm planning to add the git package later, but the contribution doc said i should only add one package at a time.

Reply:
Should I add the `ibus-bamboo-git` package to this PR? I have it ready it's template file in my local repo.

Reply:
We usually don't merge `-git` packages, only in very special edge cases, but I don't see this being one.

Reply:
oh, okay, i'll remove the `conflicts` line then